java/src/jmri/jmrix/lenz/xnetsimulator/XNetSimulatorAdapter.java

```diff
@@ -483,18 +493,41 @@ private XNetReply accInfoReply(XNetMessage m) {
            reply.setParity();
         return reply;
     }
+    
+    protected int lengthOfByteStream(XNetReply reply) {
+        return (reply.getElement(0) & 0x0f) + 2;
+    }
 
+    protected int addHeaderToOutput(byte[] msg, XNetReply m) {
+        return 0;
+    }
+
+    /**
+     * Add trailer to the outgoing byte stream.
+     *
+     * @param msg    the output byte stream
+     * @param offset the first byte not yet used
+     */
+    protected void addTrailerToOutput(byte[] msg, int offset, XNetReply m) {
```

Comment:
XpressNet messages should not have a trailer of any kind.  All XpressNet messages are binary, so the 0x0d You are adding to non-binary messages will never be added.

Reply:
Thanks, a stupid merge error when pulling (I though) minimal changes to the PR. Luckily all XpressNets are binary, but must remove this.